tests/fixtures/MakeEntityWithNotFluentSetters/tests/GeneratedEntityTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
```

Comment:
I'm not familiar with the test for this bundle, but I can't figured out where this file is used (The others fixtures files are used in \Symfony\Bundle\MakerBundle\Tests\Maker\*).

Reply:
Yes I forgot about test, added.